### DISCUSSION TITLE
[Proposal] Multi-Language Support (MLS) Team

### DISCUSSION THREAD

**Discussion by cristianMarcial** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#146] [Ideas]
During the first few weeks of this semester, _**we tried to work on the functionality to change the language of the application to make it accessible to both English and Spanish speakers**_, but we put it on hold for a while to work on more essential and primary features such as the structure of the application, and we archived what we had done with MLS of the original repo.

Since this feature had been postponed for the last milestone, I think it would be a good idea for this feature to have its own separate team. Here are my arguments:

- MLS converts many features, and all of them could not be done in a single task (as seen in the last milestone in the archived repository developed by Ray and I, MSL required a clear structure of files and directories to be able to display the strings in both languages ​​in all the screen modules of the application).

- This is a feature that distinguishes us from other teams in that it has nothing to do with the functionalities related to the elements of our application (such as Database, Profile, Login, etc.), but rather it has to do with other functionalities such as managing directories, run-time file export during execution, etc. (which are similar functionalities to those that the devs of the Curriculum team worked on).

- Comply with the Domain: If we are creating an application that seeks to replace the current enrollment system, our application must be accessible to students, that is, students/users are part of our domain and therefore the domain includes Spanish-speaking students.

**Comment by keisanabria** [comment]
I am up for contributing to this team whilst being in the Documentation team as well.

**Comment by cristianMarcial** [comment]
@cristianramos9 I'd recommend you to join this team as we'll also be working with parsing tool here which is something you have more experience with compared to other functionalities of the past milestone.

**Reply by cristianramos9** [reply]
Count me in on the team.
